tests/workbenches/notebook-controller/conftest.py

```diff
@@ -0,0 +1,282 @@
+            "coveredResources": ["cpu", "memory"],
+            "flavors": [
+                {
+                    "name": flavor_name,
+                    "resources": [
+                        {"name": "cpu", "nominalQuota": cpu_quota},
+                        {"name": "memory", "nominalQuota": memory_quota},
+                    ],
+                }
+            ],
+        }
+    ]
+
+
+@pytest.fixture(scope="class")
+def kueue_notebook_resource_flavor(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
+) -> Generator[ResourceFlavor, Any, Any]:
+    """Create a ResourceFlavor for notebook workloads"""
+    with create_resource_flavor(
+        client=admin_client,
+        name=request.param["name"],
+        teardown=True,
+    ) as resource_flavor:
+        yield resource_flavor
+
+
+@pytest.fixture(scope="class")
+def kueue_notebook_cluster_queue(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
+) -> Generator[ClusterQueue, Any, Any]:
+    """Create a ClusterQueue for notebook workloads"""
+    resource_groups = kueue_resource_groups_for_notebooks(
+        flavor_name=request.param["resource_flavor_name"],
+        cpu_quota=request.param["cpu_quota"],
+        memory_quota=request.param["memory_quota"],
+    )
+
+    with create_cluster_queue(
+        client=admin_client,
+        name=request.param["name"],
+        resource_groups=resource_groups,
+        namespace_selector=request.param.get("namespace_selector", {}),
+        teardown=True,
+    ) as cluster_queue:
+        yield cluster_queue
+
+
+@pytest.fixture(scope="class")
+def kueue_notebook_local_queue(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
+    kueue_enabled_notebook_namespace: Namespace,
+) -> Generator[LocalQueue, Any, Any]:
+    """Create a LocalQueue for notebook workloads"""
+    with create_local_queue(
+        client=admin_client,
+        name=request.param["name"],
+        cluster_queue=request.param["cluster_queue"],
+        namespace=kueue_enabled_notebook_namespace.name,
+        teardown=True,
+    ) as local_queue:
+        yield local_queue
+
+
+@pytest.fixture(scope="function")
+def kueue_notebook_persistent_volume_claim(
+    request: FixtureRequest,
+    kueue_enabled_notebook_namespace: Namespace,
+    unprivileged_client: DynamicClient,
+) -> Generator[PersistentVolumeClaim, Any, Any]:
+    """Create a PersistentVolumeClaim in the Kueue-enabled namespace"""
+    with PersistentVolumeClaim(
+        client=unprivileged_client,
+        name=request.param["name"],
+        namespace=kueue_enabled_notebook_namespace.name,
+        label={Labels.OpenDataHub.DASHBOARD: "true"},
+        accessmodes=PersistentVolumeClaim.AccessMode.RWO,
+        size="10Gi",
+        volume_mode=PersistentVolumeClaim.VolumeMode.FILE,
+    ) as pvc:
+        yield pvc
+
+
+@pytest.fixture(scope="class")
+def kueue_enabled_notebook_namespace(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
```

Comment:
admin client is only used if unprivileged client is not passed. So there is no need to pass both. If you want to use unprivileged client, then only pass it, else pass admin client.